### PR TITLE
add tauri-vue-template

### PR DESCRIPTION
**Pull request by Uninen**
This is the link to the project: [tauri-vue-template](https://github.com/Uninen/tauri-vue-template)

- [x] **I have read the [contributing guidelines](contributing.md).**
- [x] Use the following format: `[Title](link) - Description.`
- [x] Add entries to the bottom of the correct category.
- [x] No unnecessary words like `for Tauri`, `a Tauri plugin` and `Super-Fast`.
- [x] Description does not start with `A` or `An`.
- [x] Description is under 24 words.
- [x] Description has no links or parenthesis.
- [x] Use `backticks` when mentioning package names.
- [x] Double-check spelling and grammar.
- [x] No trailing whitespace is added to the end of any file.
- [x] One suggestion per pull request.
- [x] I understand that my entry will be removed if it violates the guidelines.
- [x] I have [signed my commits](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits).

### Templates

- [x] Works with **Tauri 1.x and onward**.
- [x] The repo is at least 30 days old.
- [x] Documentation is in English.
- [x] The template provides enough information about how to get started and what's included.
- [x] The template is pretty different from the existing templates.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,8 +19,9 @@ A curated collection of the best stuff from the Tauri ecosystem and community.
 
 ## Templates
 - [tauri-svelte-template](https://github.com/probablykasper/tauri-svelte-template) - Svelte template with cross-platform GitHub action builds, Vite, TypeScript, Svelte Preprocess, hot module replacement, ESLint and Prettier.
-- [tauri-react-template](https://github.com/oSethoum/tauri-react-template) - React template with Vite, TypeScript, hot module replacement.
+- [tauri-react-template](https://github.com/oSethoum/tauri-react-template) - React template with Vite, TypeScript, hot module replacement.
```
Comment (JonasKruckenberg) [inline README.md]: Can you get rid of this extra space here?

**Review by JonasKruckenberg — CHANGES_REQUESTED** [review]

**Comment by JonasKruckenberg** [comment]
Few things of note:
- You didn't set `beforeDevCommand` and `beforeBuildCommand`
- `Menu` now implements `Default` (last minute 1.0 change) so you can replace your default menu with the standard one.
- Since you advertise this as one of your templates features I would like to see more GH workflows implemented.

File: README.md (hunk 1 above)
Comment (Uninen) [reply README.md]: Sure thing 👍

**Comment by Uninen** [comment]
Thanks for the tips @JonasKruckenberg! I'll add those commands and fix the menu 👍 

I'm working on the repo as we speak, I just opened a PR regarding the GH workflows using `tauri-apps/tauri-action` to attach binaries to a GH release. I'd definitely appreciate a review for those changes if you want to keep this pending for a while more.

**Comment by Uninen** [comment]
PS. The search on Tauri homepage does not find anything with `beforeDevCommand` or `beforeBuildCommand`. (DDG does, no worries).

**Comment by FabianLars** [comment]
yeah it doesn't index the config properties yet

**Comment by Uninen** [comment]
Okay, I thought I was missing those commands but they're just empty bc the app doesnt need them; `pnpm build` builds the app fine, I don't need anything extra. (I also tested the previous version w/ GH actions and the [build works on CI](https://github.com/Uninen/tauri-vue-template/runs/6185822379?check_suite_focus=true), too. The next iteration is being built in [PR #9](https://github.com/Uninen/tauri-vue-template/pull/9))

**Comment by Uninen** [comment]
Hey @JonasKruckenberg, I just merged the GH Actions PR, there's now a fully automated CI workflow for the releases w/ binaries as well as tests (+ test builds) 👍 

I also fixed the menu to use `Menu::os_default`. Did not touch `beforeDevCommand` or `beforeBuildCommand` as there is no need for them in this setup -- but they're there for userland if someone needs them.

I'd also want to say Big Thank You and Kudos for the whole Tauri team for the 1.0 release -- it's fantastic to see true alternatives for Electron as open source! 🙏

**Comment by JonasKruckenberg** [comment]
Ah sorry, I didn't see you were using `vite-tauri` lgtm then 👍

**Review by JonasKruckenberg — APPROVED** [review]

**Pull request merged**